blocks/product-list-page/product-list-page.css

```diff
@@ -20,6 +20,22 @@
     width: 100%;
 }
 
+.product-discovery-product-actions {
+    display: flex;
+    flex-direction: row;
+}
+
+.product-discovery-product-actions__add-to-cart button {
+    width: 150px;
+    margin: 0 -10px;
+    padding: var(--spacing-xsmall) var(--spacing-xxsmall);
+
+}
+
+.product-discovery-product-actions__wishlist-toggle button {
+    width: 50px;
+}
```

Comment:
I don't see any action buttons in the PLP so I cannot confirm that this does anything.

I think this sort of CSS, which is used as integration code such as slots, is fine.